sass/materialize.scss

```diff
@@ -1,17 +1,16 @@
 @charset "UTF-8";
 
-// Mixins
-// @import "components/prefixer";
-@import "components/mixins";
-@import "components/color";
-
-// Variables;
+// 1. Variables
 @import "components/variables";
 
-// Reset
+// 2. Reset
 @import "components/normalize";
 
-// components
+// 3. Mixins
+@import "components/prefixer";
```

Comment:
The _prefixer.scss was commented out for a reason... :) Honestly I forgot why, but there was a reason!